content/docs/guides/first_steps.md

```diff
@@ -0,0 +1,175 @@
+---
+title: First steps
+sort_rank: 3
+---
+
+# First steps with Prometheus
+
+Welcome to Prometheus! Prometheus is a monitoring platform that collects metrics from monitored targets by scraping metrics HTTP endpoints on these targets. This guide will show you to how to install, configure and monitor our first resource with Prometheus. You'll download, install and run Prometheus. You'll also download and install an exporter, tools that expose time series data on hosts and services. Our first exporter will be the Node Exporter, which exposes host-level metrics like CPU, memory, and disk. 
+
+## Downloading Prometheus
+
+[Download the latest release](/download) of Prometheus for your platform, then
+extract it:
+
+```language-bash
+tar xvfz prometheus-*.tar.gz
+cd prometheus-*
+```
+
+The Prometheus server is a single binary called `prometheus` (or `prometheus.exe` on Microsoft Windows). We can run the binary and see help on its options by passing the `--help` flag.
+
+```language-bash
+./prometheus --help
+usage: prometheus [<flags>]
+
+The Prometheus monitoring server
+
+. . .
+```
+
+Before starting Prometheus, let's configure it. 
+
+## Configuring Prometheus
+
+Prometheus configuration is [YAML](http://www.yaml.org/start.html). The Prometheus download comes with a sample configuration in a file called `prometheus.yml` that's a good place to get started.
```

Comment:
Missing a . after `prometheus.yml`,

Reply:
Not sure I understand? That's one sentence.

Reply:
oh yes my bad